content/general/kafka-connector.textile

```diff
@@ -8,15 +8,74 @@ languages:
   - none
 jump_to:
   Help with:
+    - Data from Ably to Kafka#data-ably-kafka
+    - How does Ably Kafka Connector work?#how
+    - Mapping options#mapping-options
+    - Publishing messages with a schema#schema
     - Install the connector#install
     - Configure the connector#configure
     - Tutorials#tutorials
     - See also#see-also
 ---
 
-Use the Ably Kafka Connector to send data from one or more "Kafka topics":https://developer.confluent.io/learn-kafka/apache-kafka/topics/ into Ably "channels":/realtime/channels.
+The Ably Kafka Connector provides a ready-made integration between "Kafka":https://kafka.apache.org/ and Ably. It allows for realtime event distribution from Kafka to web, mobile, and IoT clients, over Ably's feature-rich, multi-protocol pub/sub channels. 
 
-If you need to send data from Ably to Kafka instead, use a "Kafka rule":/general/firehose/kafka-rule.
+**Note:** This connector is verified by Confluent as "Gold":https://www.confluent.io/hub/ably/kafka-connect-ably, following the guidelines set forth by Confluent's "Verified Integrations Program":https://www.confluent.io/verified-integrations-program/.
+
+You can use the Ably Kafka Connector to send data from one or more "Kafka topics":https://developer.confluent.io/learn-kafka/apache-kafka/topics/ into Ably "channels":/realtime/channels. The following diagram illustrates a typical deployment:
+
+<a href="/images/diagrams/ably-kafka-connector-overview.png" target="_blank">
+  <img src="/images/diagrams/ably-kafka-connector-overview.png" style="width: 100%"
+  title="Click to enlarge" alt="Ably Kafka Connector overview">
+</a>
+
+h2(#data-ably-kafka). Data from Ably to Kafka
+
+If you need to send data from Ably to Kafka, use a "Kafka rule":/general/firehose/kafka-rule.
```

Comment:
It seems wrong to have separate heading for something that we're saying use a different tool for. IMO this should be at the top of the page, to stop anyone reading further if they're looking to do Ably->Kafka rather than Kafka->Ably.

Previously this was mentioned after the "The Ably Kafka Connector provides a ready-made-integration..." sentence and IMO would be the best place for it. And maybe use some sort of callout to make it clearer?

Reply:
Yes, I think we need to call it out somehow, but perhaps a new section with heading is too much. I did this because I was asked to add this info (even though it was already on the page - it was not standing out enough!)